### PR TITLE
Add FreeList!(Buffer!(void)) tests

### PR DESCRIPTION
**Pull request by mihails-strasuns-sociomantic**
Make sure this key piece of functionality works

### CODE REVIEW

File: src/ocean/util/container/pool/FreeList.d [hunk 1]
```diff
@@ -312,6 +317,36 @@ version ( UnitTest )
...
     }
 
+    /***************************************************************************
+
+        Buffer free list tester.
+
+    ***************************************************************************/
+
+    alias FreeList!(Buffer!(void)) BufferFreeList;
+    class BufferFreeListTester : FreeListTester!(BufferFreeList.ItemType)
```
Comment (mathias-lang-sociomantic) [inline src/ocean/util/container/pool/FreeList.d]: It looks like those types, albeit hidden being `version (UnitTest)`, are public ?

Comment (mihails-strasuns-sociomantic) [reply src/ocean/util/container/pool/FreeList.d]: Good catch. Won't make a difference for D1 but should fix anyway.

**Comment by mihails-strasuns-sociomantic** [comment]
Updated with protection attributes

**Review by mathias-lang-sociomantic — APPROVED** [review]

**Comment by mathias-lang-sociomantic** [comment]
Flakey scheduler coverage ?

**Comment by mihails-strasuns-sociomantic** [comment]
@mathias-lang-sociomantic it _think_ it was just missing rebase - but will keep an eye if it repeats again

**Comment by mihails-strasuns-sociomantic** [comment]
@sociomantic-tsunami/core-team merge ping?

**Review by gavin-norman-sociomantic — APPROVED** [review]

**Pull request merged**